pkg/kubelet/metrics/metrics.go

```diff
@@ -924,7 +924,7 @@ var (
 		},
 	)
 
-	// TopologyManagerAdmissionDuration is a Histogram that tracks the duration (in seconds) to serve a pod admission request.
+	// TopologyManagerAdmissionDuration is a Histogram that tracks the duration (in milliseconds) to serve a pod admission request.
```

Comment:
oof.  We should not be using milliseconds in metric names, and definitely shouldn't be using abbreviations ("ms").

Looks like https://github.com/kubernetes/kubernetes/pull/115590 was added without sig-instrumentation review :(.  The metric was correct in the KEP: https://github.com/kubernetes/enhancements/pull/3745/files#diff-ad68be2b92ea13896be1e64d44532c326099bfde01efd369b7d16a8a85e1d4fdR55, but wasn't properly implemented.

@SergeyKanzhelev we should probably migrate this to a proper name.